### ISSUE TITLE
Provide a warning message if coala ran on nothing i.e no files

### ISSUE THREAD

**Issue by ishanSrt** [coala/coala#5079]
status: blocked by https://github.com/coala/coala/issues/4928

**Comment by ishanSrt** [comment]
would like to be assigned to this

**Comment by somnathrakshit** [comment]
@Makman2 Could you please invite me to the org and assign me this issue?

**Comment by Makman2** [comment]
@somnathrakshit This issue is currently blocked as it requires another issue to be solved :) @ishanSrt Is responsible for that issue, and because of this I assigned him here too because he knows exactly what to do.

I hope you don't mind when I keep him assigned, feel free to start with another newcomer issue ;)
(Sorry btw for commenting so late, I'm on vacation :smile:)

**Comment by ishanSrt** [comment]
I would like to be reassigned to this.
